google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/Util.java

```diff
@@ -197,4 +199,18 @@ static void recordMetricsFromMetadata(
     // Record gfe metrics
     tracer.recordGfeMetadata(latency, throwable);
   }
+
+  static GrpcCallContext addHandlerToCallContext(
+      ApiCallContext context, GrpcResponseMetadata responseMetadata, BigtableTracer tracer) {
+    if (context instanceof GrpcCallContext) {
+      // context should always be an instance of GrpcCallContext. Sanity check just in case.
```

Comment:
If this should never happen, I think we should throw an exception not swallow the error so that we can fix our assumptions